### ISSUE TITLE
Running pyLeTOH prevents SailishOS from clearing previous events from the same app from notification

### ISSUE THREAD

**Issue by datakurre** [datakurre/harbour-pyletoh#1]
To reproduce:
1. start pyLetoh
2. leave it running background
3. cause multiple notifications e.g. from Whatsapp

Expected behavior:

Notification center shows only one notification.

Resulted behavior:

Notification center shows three notifications.

**Comment by datakurre** [comment]
@kimmoli Have you seen this while eavasdropping d-bus notifications?

My d-bus code does not do much, so if you haven't seen when eavasdropping d-bus from cpp-code, this is probably just a bit unsupported use-case for d-bus [QML-bindings](https://github.com/nemomobile/nemo-qml-plugin-dbus/blob/master/src/declarativedbusadaptor.cpp#L341). I don't expect you to be familiar with that one :)

```
DBusAdaptor {
  id: interceptor
  iface: 'org.freedesktop.Notifications'
  path: '/org/freedesktop/Notifications'
  function rcNotify(app_name, replaces_id, app_icon, summary,
                    body, actions, hints, expire_timeout) {
     python.call('letoh.turn_on');
  }
}
DBusInterface {
  id: notifications
  service: 'org.freedesktop.Notifications'
  iface: 'org.freedesktop.Notifications'
  path: '/org/freedesktop/Notifications'
  signalsEnabled: true
  function notificationClosed(id, reason) {
     python.call('letoh.turn_off');
  }
}
DBusInterface {
  id: dbus
  service: 'org.freedesktop.DBus'
  path: '/org/freedesktop/DBus'
  iface: 'org.freedesktop.DBus'
}
Component.onCompleted: {
  dbus.call(
      'AddMatch',
      "interface='org.freedesktop.Notifications',member='Notify',type='method_call',eavesdrop='true'");
}
```

**Comment by datakurre** [comment]
This must be a side-effect from using QML d-bus. I don't see this, when I use the python bindings.

**Comment by kimmoli** [comment]
It somehow breaks the replaces_id ?
when publishing a new notification, notificationManager replies with an id.
When publishing another with same id, it will dismiss previous with same id.

**Comment by datakurre** [comment]
Fixed in 0.4.0 by replacing d-bus QML bindings with python daemon.
